#!/usr/bin/env python
PACKAGE = "interface"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("Joint_1", double_t, 0, "Angel (degrees)",    -60 , -360, 360)
gen.add("Joint_2", double_t, 0, "Angel (degrees)",    -60 , -180, 0)
gen.add("Joint_3", double_t, 0, "Angel (degrees)",    -60 , -180, 180)
gen.add("Joint_4", double_t, 0, "Angel (degrees)",    -60 , -360, 360)
gen.add("Joint_5", double_t, 0, "Angel (degrees)",    -60 , -360, 360)
gen.add("Joint_6", double_t, 0, "Angel (degrees)",    -60 , -360, 360)

gen.add("Activate",   bool_t,   0, "A Boolean parameter",  True)
gen.add("Execute",   bool_t,   0, "A Boolean parameter",  False)


exit(gen.generate(PACKAGE, "interface", "joint_controller"))